content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: blog
+title: "Breaking changes related to CNI version in popular container runtimes"
```

Comment:
At this point, there shouldn't be any breaking changes (now that we've fixed the libcni regression).